pkg/scripts/issues/README.md

```diff
@@ -0,0 +1,11 @@
+1. To use the script, generate access token here: https://github.com/settings/tokens?type=beta.
+2. To get all open issues invoke the [first script](./gh/main.go) setting `SF_TF_SCRIPT_GH_ACCESS_TOKEN`:
+```shell
+  cd gh && SF_TF_SCRIPT_GH_ACCESS_TOKEN=<YOUR_PERSONAL_ACCESS_TOKEN> go run .
+```
+3. File `issues.json` should be generated in the `gh` directory. This is the input file for the second script.
```

Comment:
why can't steps 2/3/4 be combined into one step? Getting and processing at same time.

Reply:
The reason was that the data does not change between runs: The result of the first script is always the same. So this way, I was able to experiment on the static data that was faster and without locking my GH account because of the API limits they have.